README.md

```diff
@@ -1,2 +1,26 @@
 # hw01
 Repository for homework 1
+
+### Andrew Gilbert
+
+#### Academic Interests
+
+I am a MAPSS student concentrating in political science. I am focusing on American politics, and I am specifically interested in:
+
+* Political representation
+* Public opinion
+* Political behavior
+
+#### Background
+
+I worked as a consultant at an environmental consulting firm for *four* years prior to enrolling in MAPSS. I was primarily responsible for developing strategies that enabled our clients to:
+
+1. Comply with environmental laws and regulations; and
+2. Influence the interpretation or implementation of public policies.
+
+#### Personal Interests
+
+Back when I had free time, I enjoyed doing 
+[street photography](https://en.wikipedia.org/wiki/Street_photography).
+
+![Street Photo](ARGilbert-20170912-ChicagoStreet06-84.jpg)
```

Comment:
Nice job - it sounds like you intuitively thinking through and figuring out the necessity of committing changes in your local repo, including adding the image, for them to render on Github, which is great.